src/task-1.js

```diff
@@ -1,6 +1,8 @@
 
```

Comment:
Not a critical point, but you have `if` which checks if a statement is `true` of `false` and inside this block, you return `true` for truly and `false` for falsy. Can we manage to do `isTriangle` f-n without `if` block?

Reply:
I got it!